src/app/utils/ProxifyUrl.js

```diff
@@ -12,6 +12,8 @@ const rProxyDomain = /^http(s)?:\/\/steemit(dev|stage)?images.com\//g;
 const rProxyDomainsDimensions = /http(s)?:\/\/steemit(dev|stage)?images.com\/([0-9]+x[0-9]+)\//g;
 const NATURAL_SIZE = '0x0/';
 
+export const imageProxy = () => $STM_Config.img_proxy_prefix;
```

Comment:
This should be a constant

Reply:
That is not an option.

Reply:
Is it injected after the app code is loaded? Could make it into a `getAvatarUrl(account)` that would justify its existence:)

Reply:
Is this compatible w/ the way the ProxifyUrl tests inject this global variable?

Reply:
@roadscape Tests are passing. :)